charts/radix-cluster-cleanup/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: 1.1.3
```

Comment:
Should we start at version 1.0.0?

Reply:
yes, just had to increment versions during testing in test cluster